src/Widgets/FormStepContainerWidget/components/FriendlyCaptchaComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { WidgetInstance } from 'friendly-challenge';
+import { CaptchaEndpoint, CaptchaStartMode } from "../../../../types/types";
+
+interface FriendlyCaptchaProps {
+  onChangeCaptcha: React.Dispatch<React.SetStateAction<string | null>>;
+  siteKey: string;
+  theme: string;
+  language: string | undefined;
+  startMode: CaptchaStartMode;
+  endpoint: CaptchaEndpoint;
+}
+const GLOBAL_ENDPOINT = "https://api.friendlycaptcha.com/api/v1/puzzle";
```

Comment:
`"https://api.friendlycaptcha.com/api/v1/puzzle";` 

Hmm maybe having an option to change it would be benneficial. I think if you do something like 

```
const GLOBAL_ENDPOINT = proccess.env.REACT_SN_FORM_FRIENDLY_CAPTCHA_GLOBAL_URL || "https://api.friendlycaptcha.com/api/v1/puzzle";
```

It might work. Something to think about. So the dev can specify different endpoint if needed before we update the package

Reply:
Unfortunately the form package does not have access to `proccess.env` variables. Already tried to receive the tenant that way. This won`t work :(